docs/scene_walkthrough/P2P_netcode_walkthrough.md

```diff
@@ -11,7 +11,7 @@ This demo showcases the Peer 2 Peer interface working with Unity's netcode. This
 
 The P2P Netcode Sample Scene is included in the `Extra Pack 1` sample pack.  
 
-Upon import, the `com.unity.netcode.gameobjects` package that this scene requires will be automatically installed. You will still need to <a href="/readme.md#importing-the-samples">add it in the build settings</a>.   
+Upon import, the `com.unity.netcode.gameobjects` package that this scene requires will be automatically installed. You will still need to <a href="/README.md#importing-the-samples">add it in the build settings</a>.   
```

Comment:
This does not lead to the "Importing the samples" section. It just leaves me at the top of the readme.